### ISSUE TITLE
unable to install 'node-red-contrib-infocache'

### ISSUE THREAD

**Issue by ent-adm** [TotallyInformation/node-red-contrib-infocache#3]
Hi,
I was trying to install the infocache node via npm as well as the admin interface but none worked.
the npm install mentioned "not found"

Is this node still available some how?

Thanks!

**Comment by antonio1000homens** [comment]
same here

**Comment by TotallyInformation** [comment]
Sorry, this is still in development. There are lots of edge-cases to consider.

**Comment by julesrulez96** [comment]
Still waiting, no change?

Thanks

**Comment by TotallyInformation** [comment]
Yes, sorry, lots of other things happening. Don't forget though that you can do the same thing with a fairly simple function node. Examples are provided with the uibuilder node and on the uibuilder WIKI.
